modules/core/src/v2/types.ts

```diff
@@ -14,3 +16,13 @@ export interface RequestTracer {
   inc(): void;
   toString(): string;
 }
+
+
+export class Types {
```

Comment:
Talked with Tyler, better if we move this to basecoin
https://github.com/BitGo/BitGoJS/blob/db79e9500298563586a1febe1820ed245dda8fd7/modules/core/src/v2/baseCoin.ts#L4